examples/latency_attributes.rs

```diff
@@ -0,0 +1,22 @@
+use web_audio_api::context::{AudioContext, BaseAudioContext};
+use web_audio_api::node::{AudioNode, AudioScheduledSourceNode};
+
+fn main() {
+    println!("AudioContextLatencyCategory::Interactive");
+    let context = AudioContext::new(Default::default());
+
+    let sine = context.create_oscillator();
+    sine.frequency().set_value(200.);
+    sine.connect(&context.destination());
+
+    sine.start();
+
+    println!("- BaseLatency: {:?}", context.base_latency());
+    println!("+ OutputLatency: {:?}", context.output_latency());
+
```

Comment:
Might be nice to just infinitely `loop` this block? Then you can properly it changing slightly over time

Reply:
right, that's done!